src/core/load_balancing/weighted_round_robin/weighted_round_robin.cc

```diff
@@ -86,6 +88,41 @@ namespace {
 
 constexpr absl::string_view kWeightedRoundRobin = "weighted_round_robin";
 
+constexpr absl::string_view kMetricLabelLocality = "grpc.lb.locality";
+
+const auto kMetricRrFallback =
+    GlobalInstrumentsRegistry::RegisterUInt64Counter(
+        "grpc.lb.wrr.rr_fallback",
+        "EXPERIMENTAL.  Number of scheduler updates in which there were not "
+        "enough endpoints with valid weight, which caused the WRR policy to "
+        "fall back to RR behavior.",
+        "{updates}", {kMetricLabelTarget}, {kMetricLabelLocality}, false);
```

Comment:
let's follow the same convention here as opentelemetry https://opentelemetry.io/docs/specs/semconv/general/metrics/#instrument-units . s/{updates}/{update}

Reply:
Done.